src/componentLibrary/NumberInput/NumberInput.style.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+import { Input, InputProps } from 'componentLibrary/Input';
+
+const BaseNumberInput = styled(Input).attrs<InputProps>({
+  type: 'number',
+  step: 'any',
+  pattern: '[-+]?[0-9]*[.,]?[0-9]+',
+  placeholder: '0.00',
+  spellCheck: 'false'
+})``;
```

Comment:
@crypto-engineer do we need those backticks?

Reply:
Let me remove it. I thought they were necessary. Thanks.

Reply:
Ok, no worries :) can't see what they're doing, but if the compiler's complaining then we'll have to live with it!